Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -237,9 +237,36 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
          * @protected
          */
         _uiSetConfirmButtonState: function () {
-            var confirmButton = this.get('container').one('.ez-universaldiscovery-confirm');
+            var confirmButton = this.get('container').one('.ez-universaldiscovery-confirm'),
+                isSelectable = this.get('isSelectable');
 
-            confirmButton.set('disabled', !this.get('selection'));
+            if (this.get('multiple') && this.get('selection')) {
+                confirmButton.set('disabled', false);
+            } else {
+                if (this.get('selection') && isSelectable(this.get('selection'))) {
```

Comment:
actually `isSelectable` should not be called to set the state of the button, it should rather be called to accept or deny the value to put in the `selection` attribute